cmd/ical-relay/config.go

```diff
@@ -195,33 +170,16 @@ func (c Config) importToDB() {
 		notifier.Name = name
 		log.Debug("Importing notifier " + name)
 		// Write the notifier to the db, adding recipients afterwards
-		dbWriteNotifier(notifier)
+		database.DbWriteNotifier(notifier)
 		for _, recipient := range notifier.Recipients {
-			dbAddNotifierRecipient(notifier, recipient)
+			database.DbAddNotifierRecipient(notifier, recipient)
 		}
 	}
 }
 
-func reloadConfig() error {
-	// load config
-	var err error
-	conf, err = ParseConfig(configPath)
-	if err != nil {
-		return err
-	} else {
-		conf.importToDB()
-		log.Info("Config reloaded")
-		return nil
-	}
-	//TODO: clear caches
-}
-
-// CONFIG EDITING FUNCTIONS
+// CONFIG DataStore FUNCTIONS
```

Comment:
should this use the Datastore Interface?

Reply:
and if so, should maybe data.go be moved back to the main package, and only the dbDataStore.go interface sit in the database package?

Reply:
this would cause in circular dependencies which I prevented by doing this. We might also want to use the DataStore interface in the notifier. Hence, I'd recommend creating an additional package called "libs" or something similar and moving everything from data.go there.